src/fides/api/ops/tasks/__init__.py

```diff
@@ -41,7 +40,7 @@ def get_new_session(self) -> ContextManager[Session]:
         return self._sessionmaker()
 
 
-def _create_celery(config: FidesConfig = get_config()) -> Celery:
+def _create_celery(config: FidesConfig) -> Celery:
```

Comment:
Nit: It'd be good to leave this defaulting to `CONFIG` here

Reply:
agreed, will fix